### ISSUE TITLE
Cannot recognize my air75

### ISSUE THREAD

**Issue by Aurawn2** [donn/nudelta#26]
Hello ,the AppImage cannot recognize my air75 on debian linux. Do I need to do anything else besides connect my usb?
![image](https://user-images.githubusercontent.com/81619245/215366244-0ac50159-266f-4977-8e74-a5bbab068993.png)

**Comment by ChristianCiach** [comment]
Same issue. Console output:

```
> ./nudelta-amd64.AppImage 
(node:13787) UnhandledPromiseRejectionWarning: ReferenceError: clipboard is not defined
    at sendKeyboardInfo (/tmp/.mount_nudeltb5FpL3/resources/app.asar/index.cjs:196:17)
    at IpcMainImpl.<anonymous> (/tmp/.mount_nudeltb5FpL3/resources/app.asar/index.cjs:205:41)
    at IpcMainImpl.emit (node:events:527:28)
    at EventEmitter.<anonymous> (node:electron/js2c/browser_init:165:11237)
    at EventEmitter.emit (node:events:527:28)
(Use `nudelta --trace-warnings ...` to show where the warning was created)
(node:13787) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:13787) UnhandledPromiseRejectionWarning: ReferenceError: clipboard is not defined
    at sendKeyboardInfo (/tmp/.mount_nudeltb5FpL3/resources/app.asar/index.cjs:196:17)
    at click (/tmp/.mount_nudeltb5FpL3/resources/app.asar/index.cjs:114:29)
    at MenuItem.click (node:electron/js2c/browser_init:73:1906)
    at a._executeCommand (node:electron/js2c/browser_init:81:2453)
(node:13787) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
```

**Comment by donn** [comment]
For now, here's a workaround: just run these in the terminal:

```sh
echo 'SUBSYSTEMS==\"usb\", ATTRS{idVendor}==\"05ac\", ATTRS{idProduct}==\"024f\", TAG+=\"uaccess\"' | sudo tee /etc/udev/rules.d/70-nudelta.rules
sudo udevadm control --reload-rules && sudo udevadm trigger
```

**Comment by donn** [comment]
Fixed in 0.6.5
